src/shared/modules/udc/udcDuck.js

```diff
@@ -50,13 +50,34 @@ export const CLEAR_EVENTS = `${NAME}/CLEAR_EVENTS`
 export const UPDATE_SETTINGS = `${NAME}/UPDATE_SETTINGS`
 export const UPDATE_DATA = `${NAME}/UPDATE_DATA`
 export const BOOTED = `${NAME}/BOOTED`
+export const METRICS_EVENT = `${NAME}/METRICS_EVENT`
 
-// Local variables (used in epics)
 let booted = false
+
+// Event constants
+export const EVENT_CLIENT_START = 'EVENT_CLIENT_START'
+export const EVENT_BROWSER_SYNC_LOGOUT = 'EVENT_BROWSER_SYNC_LOGOUT'
+export const EVENT_BROWSER_SYNC_LOGIN = 'EVENT_BROWSER_SYNC_LOGIN'
+export const EVENT_CONNECT = 'EVENT_CONNECT'
+
 const typeToEventName = {
   [CYPHER]: 'cypher_attempts',
   [CYPHER_SUCCEEDED]: 'cypher_wins',
-  [CYPHER_FAILED]: 'cypher_fails'
+  [CYPHER_FAILED]: 'cypher_fails',
+  [EVENT_CLIENT_START]: 'client_starts'
+}
+
+export const typeToMetricsObject = {
+  [CYPHER]: { category: 'cypher', label: 'executed' },
+  [CYPHER_SUCCEEDED]: { category: 'cypher', label: 'succeeded' },
+  [CYPHER_FAILED]: { category: 'cypher', label: 'failed' },
+  [EVENT_CLIENT_START]: { category: 'client', label: 'started' },
```

Comment:
Could we use `{ category: 'driver' ...}` to be more specific? "client" could be the client application, or a library, or whatever